[2024-09-09T21:26:09.785+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-09T21:26:09.804+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.etl_process manual__2024-09-09T21:26:06.750230+00:00 [queued]>
[2024-09-09T21:26:09.814+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.etl_process manual__2024-09-09T21:26:06.750230+00:00 [queued]>
[2024-09-09T21:26:09.814+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-09T21:26:09.830+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): etl_process> on 2024-09-09 21:26:06.750230+00:00
[2024-09-09T21:26:09.835+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=517) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-09T21:26:09.837+0000] {standard_task_runner.py:72} INFO - Started process 518 to run task
[2024-09-09T21:26:09.837+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main', 'etl_process', 'manual__2024-09-09T21:26:06.750230+00:00', '--job-id', '220', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp4hednkfa']
[2024-09-09T21:26:09.838+0000] {standard_task_runner.py:105} INFO - Job 220: Subtask etl_process
[2024-09-09T21:26:09.887+0000] {task_command.py:467} INFO - Running <TaskInstance: main.etl_process manual__2024-09-09T21:26:06.750230+00:00 [running]> on host 1172ef8b9371
[2024-09-09T21:26:09.977+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='etl_process' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T21:26:06.750230+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-09T21:26:06.750230+00:00'
[2024-09-09T21:26:09.978+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-09T21:26:22.545+0000] {logging_mixin.py:190} INFO - 590707
[2024-09-09T21:26:22.675+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-09T21:26:22.699+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-09T21:26:22.700+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main, task_id=etl_process, run_id=manual__2024-09-09T21:26:06.750230+00:00, execution_date=20240909T212606, start_date=20240909T212609, end_date=20240909T212622
[2024-09-09T21:26:22.748+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-09T21:26:22.772+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-09T21:26:22.774+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
